src/App.js

```diff
@@ -1,15 +1,41 @@
-import SearchPage from "../src/components/SearchPage";
-import concerts from "../src/data.json";
+import SearchPage from "../src/pages/SearchPage";
+import { useState } from "react";
+import DetailPage from "./pages/DetailPage";
+import data from "../src/data.json";
+import getConcertDetails from "./services/getConcertDetails";
 
 export default function App() {
-  const pageName = "Suche";
+  const [activePage, setActivePage] = useState("suche");
+  const [concertDetails, setConcertDetails] = useState({});
```

Comment:
You can also set (null ) for concertDetails-useState :-)

Reply:
You give this state an object right? The concert which will be opened to show the details? If so, then this starting point of the state is correct 👍 and I wouldn't use `null`. 😃

Reply:
Yes, it should be the state for the object. But Danica first I was thinking about setting the start point null at this point as well 😉